### ISSUE TITLE
training code

### ISSUE THREAD

**Issue by aliasgharkhani** [MadryLab/backgrounds_challenge#6]
Hi,

I wonder is it possible for you to provide the train code with all the hyperparameters you used for training? Or the complete hyperparameters of the train (e.g., the parameters of the color jitter augmentation is not mentioned in the paper). I would appreciate it a lot if you can do that.

Thank you

**Comment by kaixiao** [comment]
Hi,

We use the following data augmentation code from a separate repository as our data augmentation. They are from the following public repository:

https://github.com/MadryLab/robustness/blob/6347646ae47120d35f47220eabd507cebbd6c914/robustness/data_augmentation.py#L40-L51

Let us know if you have any other questions.

**Comment by aliasgharkhani** [comment]
Hi,

Thank you for your response. Could you please also provide the lr drop amount on each scheduler step? In the paper you have mentioned that you drop the lr every 50 epochs, bbut you haven't mentioned the amount of drop.

Thank you

**Comment by kaixiao** [comment]
Hi,

We drop the LR by multiplying it by 0.1 every 50 epochs. We use the default setting from the public repository linked: https://github.com/MadryLab/robustness/blob/a9541241defd9972e9334bfcdb804f6aefe24dc7/robustness/defaults.py#L75

Best,
Kai

**Comment by aliasgharkhani** [comment]
Thank you
